app/controllers/application_controller.rb

```diff
@@ -5,6 +5,7 @@ class ApplicationController < ActionController::Base
 
    before_action :configure_permitted_parameters, if: :devise_controller?
 
+
```

Comment:
Why this newline had appeared here? It is important to keep version history clean and readable. Such random lines makes it hard to understand what changes had been made to particular file.